app/controllers/admin/administrators/success_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Admin
+  module Administrators
+    class SuccessController < Admin::BaseController
+      skip_after_action :verify_policy_scoped
+
+      def create
+        user = User.new(params.require(:user).permit(:full_name, :email))
```

Comment:
Same here, maybe we could have `new_user_params` method in the base controller?

Reply:
I'm not sure that belongs in the base controller?

Reply:
I guess depends how often you end up needing them in admin controllers - and how much of a complex hierarchy do you want. (and your personal tolerance for copy-pasting user params method)

Maybe we have enough user management things that they warrant a `AdminUserController < AdminController`? And the user params method would live in that new controller?